internal/api/v1/user/users/xpub_api_test.go

```diff
@@ -100,7 +100,7 @@ func TestXPubAPI_XPub(t *testing.T) {
 			transport.RegisterResponder(http.MethodGet, URL, tc.responder)
 
 			// then:
-			got, err := wallet.XPub(context.Background())
+			got, err := wallet.XPubAPI(context.Background())
```

Comment:
I've added a comment to highlight the question about the name change from the previous comment.

Reply:
it was intentional as it refer to the name of field in Client struct .. and allowing me to add method to return xPub .. 
Why this was called before XPub if is referencing some XPubAPI ? what is XPubAPI 
`res, err := c.xpubAPI.XPub(ctx)`